werkit/aws_lambda/build.py

```diff
@@ -45,10 +45,19 @@ def create_venv_with_dependencies(
 
 
 def find_site_packages_dir(venv_dir):
-    result = os.path.join(venv_dir, "lib64", "python3.7", "site-packages")
-    if not os.path.exists(result):
-        result = os.path.join(venv_dir, "lib", "python3.7", "site-packages")
-    return result
+    try:
+        return next(
```

Comment:
What is this function `next`, and where is it defined?

Reply:
Oh, I see, this is a built-in function, and this is picking out the first path that exists.

Reply:
Replaced with a simpler for loop in f78d154.